diff --git a/node_modules/prismarine-registry/index.d.ts b/node_modules/prismarine-registry/index.d.ts
index 474a3dd..6e76fad 100644
--- a/node_modules/prismarine-registry/index.d.ts
+++ b/node_modules/prismarine-registry/index.d.ts
@@ -9,6 +9,5 @@ interface PCRegistry extends IndexedData {
 interface BedrockRegistry extends IndexedData {
 
 }
-type Registry = PCRegistry & BedrockRegistry
-declare function loader(mcVersion: string): Registry
-export = loader
+export type Registry = PCRegistry & BedrockRegistry
+export default function loader(mcVersion: string): Registry
